packages/jekyll-yaml/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "jekyll yaml metadata"
+title: "Jekyll YAML metadata"
+description: A simple package to add metadata to your jekyll posts before writing.
+version: 0.1.0
+author: mathis
+tags: ["jekyll", "yaml"]
```

Comment:
Would you kindly add a `homepage:` key and value, please, even if the latter is simply https://github.com/nullnein/espanso-hub/tree/main?
We've had problems fully merging packages where a homepage: is missing.

Reply:
sure, done. This should probably be added to the Docs/the template files ;-).

Reply:
Yes, it's in progress https://github.com/espanso/website/pull/78